include/rmm/logger.hpp

```diff
@@ -96,6 +96,11 @@ struct bytes {
   }
 };
 
+inline spdlog::logger& logger()
```

Comment:
It's still possible for users to call this function even though it's in `detail`. So how does this help with simplifying dependencies?

Reply:
We no longer leak an ABI-unstable third party object out of the public api

Reply:
Yes, exactly. This function was primarily added so as to have a way to call this internally in places that need it for the moment without getting compiler warnings.